playwright/tests/request-blocking.spec.ts

```diff
@@ -0,0 +1,22 @@
+import * as selectors from '../data/selectors.json';
+import { expect, test } from '@playwright/test';
+
+test.describe('Request Blocking', () => {
+    let beforeActionVisibility;
```

Comment:
Could you please declare the type of these variables? Check the other spec files as well.

Reply:
Done, thank you!

Reply:
<img src="https://media2.giphy.com/media/nmBKiNb7h3tIv3BO8D/giphy.gif"/>